subprojects/common/time/src/main/kotlin/com/avito/time/DefaultTimeProvider.kt

```diff
@@ -12,9 +14,17 @@ import java.util.concurrent.TimeUnit
  * Used in android runtime < 26, so no java.time API yet
  * consider using ThreeTenABP
  */
-class DefaultTimeProvider : TimeProvider {
+class DefaultTimeProvider(loggerFactory: LoggerFactory) : TimeProvider {
 
-    override val timeZone: TimeZone = TimeZone.getTimeZone("Europe/Moscow")
+    private val logger = loggerFactory.create<DefaultTimeProvider>()
+
+    private val locale: Locale = Locale.getDefault()
+
+    override val timeZone: TimeZone = TimeZone.getDefault()
+
+    init {
+        logger.debug("Creating new time provider: Locale = $locale; TimeZone = $timeZone")
```

Comment:
Even debug seems too verbose. Time provider is a lightweight interface about standard classes: TimeZone, Locale, DateFormat, ... Don't see reasons to log it.

Reply:
It's in fact crucial for this PR; i want to log defaults for locale and timezone for all environments: gradle, emulators

Reply:
> i want to log defaults for locale and timezone for all environments

Ok, that seems reasonable. I propose to do it one time while starting the environment. 
I found it excessive to log the environment on each creation of this simple util.